_editorial_policies/02_reviewing.md

```diff
@@ -28,4 +18,10 @@ manuscript.
 Specific details on what reviewers look for are set out in the
 sections of the [authors guide](/authors/policies/) for each manuscript type.
 
+## Conflicts of interest
+
+A reviewer may not have conflict of interest with an author of paper
```

Comment:
Again, it's not a "conflict of interest with an author", it's a conflict of interest that might affect their ability to fairly evaluate the work of that author. Better phrasing would be: "A reviewers may not review a paper if they have a conflict of interest which would affect their ability to fairly evaluate the work, or potentially be perceived by an independent observer as affecting their ability to fairly evaluate the work." 

And did you want to say that reviewers should disclose any potential conflicts to the editors? Specifically, something like, "All reviewers are responsible for reviewing our Conflict of Interest definitions and disclosing any potential conflicts of interests to the editor handling the submission."